src/reportSources/legacyFflogs/eventAdapter/translate.ts

```diff
@@ -359,6 +358,37 @@ export class TranslateAdapterStep extends AdapterStep {
 		}
 	}
 
+	private adaptCombatantInfoEvent(event: CombatantInfoEvent): Array<Events['actorUpdate']> {
+		// TODO: Use more info in here. We're currently extracting the speed attribute values for the logging player, but there's also player level, prepull statuses, and more in there.
+
+		const foo: Array<[number | undefined, Attribute]> = [
```

Comment:
Rename variable to indicate what it's used for

Reply:
ah gdi